bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/gems/GemsMagnitudeTable.scala

```diff
@@ -102,12 +102,16 @@ object GemsMagnitudeTable extends MagnitudeTable {
     def getNominalMagnitudeConstraints(cwfs: CanopusWfs): MagnitudeConstraints
   }
 
+  // Values provided by science are at:
+  //      SB Any, CC 70, IQ 70.
+  // Add 0.8 to faintness / brightness limits to get values for the calculator, which assumes:
+  //      SB  20, CC 50, IQ 70.
   lazy val CanopusWfsMagnitudeLimitsCalculator = new CanopusWfsCalculator {
-    override def gemsMagnitudeConstraint(starType: GemsGuideStarType, nirBand: Option[MagnitudeBand]) =
-      magLimits(RBandsList, 16.3, 8.8)
+    override def gemsMagnitudeConstraint(starType: GemsGuideStarType, nirBand: Option[MagnitudeBand]): MagnitudeConstraints =
+      magLimits(RBandsList, 17.8, 11.3)
```

Comment:
Maybe we should use named constants here? specially given they are used twice

Reply:
I am a big fan of names constants... having to search for methods to figure out what their unnamed parameters mean is often too time consuming.

I like this idea, so I think I will do it for both Canopus and ODGW.